packages/core/src/pages/cp_content_schema_404.jsonc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
question: just to understand the naming convention -- those `cp_content_schema_..` would be the content types schema? Why not `cp_content_type_schema_...`? And the others `cp_section_schema`

Reply:
Yes! The prefixes are actually enforced by the CLI. 
And the reasoning behind using `cp_content_schema` and `cp_schema` was just to make it shorter 😅 . We're not using the `sections` name anymore, but we could go with `cp_component_schema_...` and `cp_content_type_schema...`. I just think that would make the names too long hahahah do you think just `cp_schema` and `cp_content_schema` would be confusing?

Reply:
> We're not using the sections name anymore

Oh, got it!

> do you think just cp_schema and cp_content_schema would be confusing?

I think it's a little confusing, because it tries to give some info but isn't clear 🤔  Depending on how this will be documented to users, `cp_schema` might be enough for the components schema, but I think that `cp_content_schema` seems weird and might be better be more clear `cp_content_type_schema` in this case. What do you think?